ui/page/components/items_scroll.go

```diff
@@ -24,6 +24,12 @@ const (
 	up   scrollDerection = 2
 )
 
+type dataList[T any] struct {
+	items    []T
+	idxStart int
```

Comment:
what is idxStart

Reply:
idxStart and edxEnd is a current index of items visible on list scroll, they are used to identify items to display in cache items